### PR TITLE
Add phpcs + ci (solves #5)

### PR DESCRIPTION
**Pull request by websupporter**
This PR adds the PHPCS ruleset as well as a .travis.yml and a bitbucket-pipelines.yml and updates the composer.json template.

solves #5

### CODE REVIEW

File: _templates/phpcs.config.xml.template [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<ruleset name="phpcs configuration">
+    <!-- As long as we run on Inpsyde .10, we need to exclude uppercase constant our self. -->
```
Comment (websupporter) [inline _templates/phpcs.config.xml.template]: We could also push for the 0.2 release https://github.com/inpsyde/php-coding-standards/commit/e150a7f3a0a008e2de5b542b08b9688be75cc22f

**Comment by dnaber-de** [comment]
@websupporter Please remember to update the CHANGELOG.md

**Comment by websupporter** [comment]
Done with https://github.com/inpsyde/boilerplate/commit/a6ef897a33f2fbf5910ce433d0bcd43b3860d9a2 @dnaber-de

**Comment by dnaber-de** [comment]
Thanks. :+1:

**Pull request merged**